src/main/java/edu/kis/vh/stacks/StackFIFO.java

```diff
@@ -1,21 +1,35 @@
 package edu.kis.vh.stacks;
 
-public class StackFIFO extends stack {
+import edu.kis.vh.stacks.collections.list.StackList;
+import edu.kis.vh.stacks.collections.list.StackVector;
+
+public class StackFIFO extends Stack {
+
+	private final StackList temp = new StackList();
```

Comment:
Proponowałbym StackVector, bo nie po to refaktoryzujemy do bridga by klasa potomna znała implementacje interfejsu chcemy to rozdzielić